src/cfnlint/data/CloudSpecs/ap-northeast-1.json

```diff
@@ -6213,6 +6287,54 @@
         }
       }
     },
+    "AWS::EC2::LaunchTemplate.CapacityReservationPreference": {
```

Comment:
This seems wrong.

Reply:
Yeah -- I'm having a heck of a time untangling this. looks like there were several undocumented property types injected into the spec around capacity.  My Saturday morning brain is having challenges patching around them.

Reply:
Yea no rush. I can help decouple it on Monday. It doesn’t look easy to decouple.

Reply:
Patched it (and also AWS::CodeBuild::Project.FilterGroup)